pkgs/tools/misc/man-db/default.nix

```diff
@@ -68,8 +68,7 @@ stdenv.mkDerivation rec {
     # make sure that we don't wrap symlinks (since that changes argv[0] to the -wrapped name)
     find "$out/bin" -type f | while read file; do
       wrapProgram "$file" \
-        --prefix PATH : "${lib.getBin groff}/bin" \
-        --prefix PATH : "${lib.getBin zstd}/bin"
+        --prefix PATH : "${lib.makeBinPath [ groff zstd ]}"
```

Comment:
This will circumvent https://github.com/NixOS/nixpkgs/pull/128503. Not sure it that is a problem.